src/server/game/Warden/WardenWin.cpp

```diff
@@ -398,51 +519,45 @@ void WardenWin::HandleData(ByteBuffer& buff)
 #endif
     }
 
-    WardenCheckResult* rs;
-    WardenCheck* rd;
-    uint8 type;
     uint16 checkFailed = 0;
 
-    ACE_READ_GUARD(ACE_RW_Mutex, g, sWardenCheckMgr->_checkStoreLock);
-
-    for (std::list<uint16>::iterator itr = _currentChecks.begin(); itr != _currentChecks.end(); ++itr)
+    for (uint16 const checkId : _CurrentChecks)
     {
-        rd = sWardenCheckMgr->GetWardenDataById(*itr);
-        rs = sWardenCheckMgr->GetWardenResultById(*itr);
-
-        type = rd->Type;
-        switch (type)
+        WardenCheck const* rd = sWardenCheckMgr->GetWardenDataById(checkId);
+        switch (rd->Type)
```

Comment:
uint8 type;
type = rd->Type;
switch (type)

Revert this switch or update IF below because we get compiling error with ENABLE_EXTRA_LOGS = 1

Reply:
seriously... you kidding me?